Controller/Adminhtml/Import/Ajax.php

```diff
@@ -39,6 +41,9 @@ public function __construct(
      */
     public function execute()
     {
+        echo json_encode(array('CUSTOMERS' => $this->customerData->getCustomers((int)$this->request->getParam('storeId')))); // GET CUSTOMER DATA
```

Comment:
it would be best if you put the `storeId` in a variable, since it will be reused